packages/file_selector/file_selector_windows/example/lib/open_text_page.dart

```diff
@@ -9,7 +9,7 @@ import 'package:flutter/material.dart';
 /// displays its contents in a dialog.
 class OpenTextPage extends StatelessWidget {
   /// Default Constructor
-  const OpenTextPage({Key? key}) : super(key: key);
+  const OpenTextPage({super.key});
```

Comment:
The `await file.readAsString();` below fails if I open a UTF-16 encoded file. Should we update this sample to handle UTF-16 since that's the most common file encoded on Windows?

Reply:
Updated. I also updated the `SaveTextPage` to actually save an Utf-16 encoded file.